### PR TITLE
Start iyarito

### PR DESCRIPTION
**Pull request by SSlayer9**

### CODE REVIEW

**Comment by SSlayer9** [comment]
sorry i forgot the ability

**Comment by tmagicturtle** [comment]
Looks good to me.

File: data/mods/ssb/moves.ts [hunk 1]
```diff
@@ -743,6 +743,26 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 		target: "normal",
 		type: "???",
 	},
+
```
Comment (ghost) [inline data/mods/ssb/moves.ts]:
```suggestion

```

File: data/mods/ssb/moves.ts [hunk 2]
```diff
@@ -743,6 +743,25 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 		target: "normal",
 		type: "???",
 	},
+	// iyatiro
```
Comment (ghost) [inline data/mods/ssb/moves.ts]:
```suggestion
	},

	// iyatiro
```

**Pull request merged**